src/tiled/selectsametiletool.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * selectsametiletool.h
+ * Copyright 2015, Mamed Ibrahimov <ibramlab@gmail.com>
+ *
+ * This file is part of Tiled.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#ifndef SELECTSAMETILETOOL_H
+#define SELECTSAMETILETOOL_H
+
+
+#include "abstracttiletool.h"
+
+#include "tilelayer.h"
+
+namespace Tiled {
+namespace Internal {
+
+class MapDocument;
+
+/**
+ * Implements a tool that bucket fills (flood fills) a region with similar tiles.
```

Comment:
This comment is still wrong, please read it carefully. This is not a fill tool. :-)

While you're at it, you may want to fix the documentation comment of the `MagicWandTool` as well...